src/plugins/application-plugin.js

```diff
@@ -0,0 +1,86 @@
+import Phaser from 'phaser';
+import { Application } from 'springroll';
+import { GAMEPLAY, SCENE, PLUGIN_NAME } from '../constants';
+
+// the easiest way to treat an Application as if it were a phaser plugin
+// is to extend Application then implement the plugin functions:
+// -- init, boot, start, stop, destroy
+// also have a PluginManager param in the constructor.
+
+/**
+ * @export
+ * @class ApplicationPlugin
+ * @extends Application
+ */
+export class ApplicationPlugin extends Application
```

Comment:
I like this (I didn't know Phaser 3 did this...)

However, I wonder if there's a better name out there since this is so similar to `springroll.ApplicationPlugin`...

Reply:
I hadn't thought of that. `SpringrollPlugin` maybe?